human_development/global_data_lab_hid_transform_load_dlt.py

```diff
@@ -14,9 +14,27 @@ def global_data_lab_hd_index():
             F.when(F.col("country_name") == 'Burkina Faso',
                 F.when(F.col("Region") == 'Boucle de Mouhoun',
                     F.lit("Boucle Du Mouhoun")
+                ).when(F.col("Region") == 'Centre (incl Ouagadougou)',
+                    F.lit("Centre")
```

Comment:
Wonder if we should update this to "Centre Region Burkina Faso" to be consistent. What do you think @nihaas21? It would also requrie harmonizing the BOOST labeling I assume?